src/application/App.tsx

```diff
@@ -127,11 +131,28 @@ function LeftSidePanelWrapper() {
 
   return (
     <>
-      <PrimarySideBar
+      <SideBar
         activePane={primaryPaneCollapsed ? null : primaryPane}
-        onClick={handleSideBarClick}
-        onSettingsClick={openSettings}
+        footerItems={[
+          // TODO: Implement Keybinds screen
+          {
+            label: 'Keybinds',
+            Icon: <MdKeyboard aria-label="Keybindings" size="24" />,
+            onClick: noop,
+          },
+          {
+            label: 'Settings',
+            Icon: <MdSettings aria-label="Settings" size="24" />,
+            onClick: openSettings,
+          },
+        ]}
+        items={[
+          { pane: 'Explorer', Icon: <FilesIcon /> },
+          { pane: 'References', Icon: <ReferencesIcon /> },
+        ]}
+        onItemClick={handleSideBarClick}
       />
+      {!primaryPaneCollapsed && <div className="h-full w-[1px]" style={{ backgroundColor: '#eff1f4' }} />}
```

Comment:
Why do we need this? Also, with that hard-coded color.

Reply:
Oops, that comes from my initial tests, when I hadn't created the tailwind variables, I forgot to update it.

Reply:
Ok. I see that this is a vertical line only visible when the primaryPanel is visible.